internal/decision/engine_test.go

```diff
@@ -104,7 +104,7 @@ func newTestEngineWithSampling(ctx context.Context, idStr string, peerSampleInte
 	e.StartWorkers(ctx, process.WithTeardown(func() error { return nil }))
 	return engineSet{
 		Peer: peer.ID(idStr),
-		//Strategy: New(true),
+		// Strategy: New(true),
```

Comment:
FYI this change is incorrect.
```go
//frux
```
and
```go
// frux
```
Are both "valid" but have different sementics. The first one is commented out code, the second one is a comment you shouldn't try to run it.

IMO commented out code is generaly a problem, we have VCS don't leave comented code it's really confusing, but that an other problem.

Reply:
yeah my editor is a bit too eager to format things sometimes